var/spack/repos/builtin/packages/rocfft/package.py

```diff
@@ -118,13 +119,14 @@ def check(self):
         "5.0.2",
         "5.1.0",
         "5.1.3",
+        "5.2.0",
     ]:
         depends_on("hip@" + ver, when="@" + ver)
         depends_on("rocm-cmake@%s:" % ver, type="build", when="@" + ver)
 
     patch("0001-Improve-compilation-by-using-sqlite-recipe-for-rocfft.patch", when="@5.0.0:5.0.2")
     patch("0002-Fix-clients-fftw3-include-dirs-rocm-4.2.patch", when="@4.2.0:4.3.1")
-    patch("0003-Fix-clients-fftw3-include-dirs-rocm-4.5.patch", when="@4.5.0:")
+    patch("0003-Fix-clients-fftw3-include-dirs-rocm-4.5.patch", when="@4.5.0:5.1.3")
```

Comment:
Same here, please comment patches. [This](https://docs.fedoraproject.org/en-US/packaging-guidelines/PatchUpstreamStatus/) is a good summary of the reason why I am asking for that (`s/Fedora/Spack/g`).

Reply:
This patch is for adding spack build test support. Added the comment in the recipe as well.